packages/mobile/src/components/image/TrackImage.tsx

```diff
@@ -62,7 +71,8 @@ export const useTrackImage = ({ track, size }: UseTrackImageOptions) => {
     cid,
     size,
```

Comment:
so just to double check, the size param is mostly irrelevant for this new path since we set the cid = the cover_art_cids[size] right?

Reply:
yes!